coveo-eks/_outputs.tf

```diff
@@ -0,0 +1,120 @@
+  value       = "${module.k8s-cluster.cluster_endpoint}"
+}
+
+output "cluster_version" {
+  description = "The Kubernetes server version for the EKS cluster."
+  value       = "${module.k8s-cluster.cluster_version}"
+}
+
+output "cluster_security_group_id" {
+  description = "Security group ID attached to the EKS cluster."
+  value       = "${module.k8s-cluster.cluster_security_group_id}"
+}
+
+output "cluster_iam_role_name" {
+  description = "IAM role name of the EKS cluster."
+  value       = "${module.k8s-cluster.cluster_iam_role_name}"
+}
+
+output "cluster_iam_role_arn" {
+  description = "IAM role ARN of the EKS cluster."
+  value       = "${module.k8s-cluster.cluster_iam_role_arn}"
+}
+
+output "workers_asg_arns" {
+  description = "IDs of the autoscaling groups containing workers."
+  value       = "${module.k8s-cluster.workers_asg_arns}"
+}
+
+output "workers_asg_names" {
+  description = "Names of the autoscaling groups containing workers."
+  value       = "${module.k8s-cluster.workers_asg_names}"
+}
+
+output "workers_user_data" {
+  description = "User data of worker groups"
+  value       = "${module.k8s-cluster.workers_user_data}"
+}
+
+output "workers_default_ami_id" {
+  description = "ID of the default worker group AMI"
+  value       = "${module.k8s-cluster.workers_default_ami_id}"
+}
+
+output "workers_launch_template_ids" {
+  description = "IDs of the worker launch templates."
+  value       = "${module.k8s-cluster.workers_launch_template_ids}"
+}
+
+output "workers_launch_template_arns" {
+  description = "ARNs of the worker launch templates."
+  value       = "${module.k8s-cluster.workers_launch_template_arns}"
+}
+
+output "workers_launch_template_latest_versions" {
+  description = "Latest versions of the worker launch templates."
+  value       = "${module.k8s-cluster.workers_launch_template_latest_versions}"
+}
+
+output "worker_security_group_id" {
+  description = "Security group ID attached to the EKS workers."
+  value       = "${module.k8s-cluster.worker_security_group_id}"
+}
+
+output "worker_iam_instance_profile_arns" {
+  description = "default IAM instance profile ARN for EKS worker groups"
+  value       = "${module.k8s-cluster.worker_iam_instance_profile_arns}"
+}
+
+output "worker_iam_instance_profile_names" {
+  description = "default IAM instance profile name for EKS worker groups"
+  value       = "${module.k8s-cluster.worker_iam_instance_profile_names}"
+}
+
+output "worker_iam_role_name" {
+  description = "default IAM role name for EKS worker groups"
+  value       = "${module.k8s-cluster.worker_iam_role_name}"
+}
+
+output "worker_iam_role_arn" {
+  description = "default IAM role ARN for EKS worker groups"
+  value       = "${module.k8s-cluster.worker_iam_role_arn}"
+}
+
+output "cluster_namespaces" {
+  description = "All namespaces created in the cluster"
+  value       = "${concat(list("infrastructure", "csec" , "monitoring", "ingress"), var.namespaces)}"
```

Comment:
Shouldnt the list be all configurable ?

Reply:
it is , we have some default namespaces (infra,csec,...) merged with the variable `namespaces` which contain all ns defined by user.